mlir/lib/Dialect/MIOpen/Transforms/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ add_mlir_dialect_library(MLIRMIOpenTransforms
   AffineTransforms.cpp
   AlignTiling.cpp
   LowerMIOpenOps.cpp
+  CopyOpt.cpp
```

Comment:
nit picking: Probably need to reorder the sequence of source code modules alphabetically.